### PR TITLE
Change PoW to 2^30 node graph.

### PR DESCRIPTION
**Pull request by dincho**
- [ ] update [Wiki](https://github.com/aeternity/epoch/wiki/Troubleshooting#node-failing-mining-attempts) after merge
- [x] trigger system tests run (EDIT: https://circleci.com/gh/aeternity/epoch/23960)

PT: https://www.pivotaltracker.com/story/show/154875588

### CODE REVIEW

File: docs/cuda-miner.md [hunk 1]
```diff
@@ -56,13 +56,13 @@ Compilation of CUDA miner is done by invoking:
 
 ```bash
 cd apps/aecuckoo && make c_src/.git
-cd c_src/src && make libblake2b.so cuda28
+cd c_src/src && make libblake2b.so cuda30
```
Comment (lucafavatella) [inline docs/cuda-miner.md]: Maybe tag `tags/v0.17.0` above shall be updated to `tags/v0.19.0` because of new cuckoo commit?

**Comment by lucafavatella** [comment]
I am taking the liberty of marking as WIP while system tests on CI server running.

File: docs/cuda-miner.md (hunk 1 above)
Comment (dincho) [reply docs/cuda-miner.md]: @lucafavatella I don't find it relevant to this PR/Commits. I can do it in separate one or in the release prep. as always.

Maybe we should start maintaining `tags/latest` ?

Comment (lucafavatella) [reply docs/cuda-miner.md]: > the release prep.

I will. Thanks.

**Pull request merged**